Resources/Prototypes/_RMC14/Entities/Structures/Walls/walls.yml

```diff
@@ -301,14 +301,6 @@
   - type: Tag
     tags:
     - Wall
-  - type: Destructible
-    thresholds:
-    - trigger:
-        !type:DamageTrigger
-        damage: 3000
-      behaviors:
-      - !type:DoActsBehavior
-        acts: [ "Destruction" ]
```

Comment:
this is for 'jungle vegetation', this was removed since they arent even damageable to begin with being invincible. are they intended to be damageable? if so should be parented to something else instead of this change

Reply:
Checked and yeah this is fine, they have a damage_cap of 3000 in cmss13 but they are also a hull wall so not actually destroyable.